pkg/cephfs/controllerserver.go

```diff
@@ -296,7 +296,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(
 // ExpandVolume expand CephFS Volumes on demand based on resizer request
 func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
 	if err := cs.validateExpandVolumeRequest(req); err != nil {
-		klog.Errorf("ControllerExpandVolumeRequest validation failed: %v", err)
+		klog.Errorf(util.Log(ctx, "ControllerExpandVolumeRequest validation failed: %v"), err)
 		return nil, err
```

Comment:
@Madhu-1 request that messages at like 322 and 329 in this function also add the logging context.

Reply:
done